src/ecrc-api/.mvn/bin/maven-wrapper.properties

```diff
@@ -0,0 +1,2 @@
+distributionUrl=https://repo.maven.apache.org/maven2/org/apache/maven/apache-maven/3.6.3/apache-maven-3.6.3-bin.zip
```

Comment:
I'm not sure that this file needs to be included.

Reply:
Good catch, added by mistake when adding the other properties files. Will remove